product-backend/src/main/java/org/kainos/ea/cli/Employee.java

```diff
@@ -0,0 +1,50 @@
+package org.kainos.ea.cli;
+
+public class Employee {
```

Comment:
I think you should have one model Employee
what do you think @leszekgu

Reply:
Story in dev notes suggest User. only reference to employee is that it's a Role value

Reply:
So we keep this User model (ex Employee), right?

Reply:
class should be User

Reply:
@leszekgu it is User. I changed all of it as I mentioned